policy-utils/Cargo.toml

```diff
@@ -11,13 +11,12 @@ std = ["serde/std", "serde_json/std", "ring", "x509-parser", "rustls", "wasi-typ
 
 [dependencies]
 err-derive = { version = "0.2", default-features = false }
-hex = { version = "=0.4.2", default-features = false }
+hex = { version = "0.4.2" }
 
-# Pinned due to Rust compiler version.
-lexical-core = { version = "=0.7.0", default-features = false }
+lexical-core = { version = "0.8.2", default-features = false }
 ring = { git = "https://github.com/veracruz-project/ring.git", version = "=0.16.11", branch = "veracruz", optional = true }
 rustls = { git = "https://github.com/veracruz-project/rustls.git", branch = "veracruz", optional = true }
 serde = { git = "https://github.com/veracruz-project/serde.git", features = ["derive"], branch = "veracruz" }
 serde_json = { git = "https://github.com/veracruz-project/json.git", branch = "veracruz", default-features = false }
-wasi-types = { git = "https://github.com/veracruz-project/wasi-types.git", branch = "veracruz" }
-x509-parser = { version = "0.7.0", optional = true }
+wasi-types = { git = "https://github.com/egrimley-arm/wasi-types.git", branch = "veracruz" }
```

Comment:
Can this be moved under `veracruz-project`?

Reply:
As above, I'm hoping https://github.com/veracruz-project/wasi-types/pull/6 can be merged first.